### PR TITLE
Complete redesign of the nanotrasen.se webpage. 

### PR DESCRIPTION
**Pull request by JorisKeir**
Still small things that need to be fixed.

- I added in the patreon banner but I didn't end up use it. It does seem like this might be a thing future devs might want to use, who knows, let me know if I got to axe it, no problem.

- I have no idea about licences or legal stuff that's required to go into the footer, so for now it's placeholder text.

- I might update the background soon to give it a more space vibe, in place of the now 100% abstract look I got going now.

EDIT: I forget a picture: 
chrome_2016-12-16_21-45-14

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -1,49 +1,157 @@
...
     text-align: center;
-    width: 100%;
+    margin: auto;
+    width: 25%;
+    clear: none;
+}
+
+
+a{
+    color: whitesmoke;
```
Comment (Regen1) [inline index.css]: User hex or rbg

File: index.css [hunk 2]
```diff
@@ -1,49 +1,157 @@
...
+    padding: 50px 0px 25px 0px;
+    font-size: 2.5em;
 }
 
-h2 .header, h3 .header {
-    height: 3px
+
+main{
+    margin:0;
+    color: white;
```
Comment (Regen1) [inline index.css]: #ffffff

File: index.css [hunk 3]
```diff
@@ -1,49 +1,157 @@
...
+main h2{
+    font-size: 1.4em;
+    padding-bottom: 20px;
+}
+main a{
+    text-decoration: 
+    underline;
+}
+main a:hover{
+    color: darkgray;
```
Comment (Regen1) [inline index.css]: Hex for color

File: index.css [hunk 4]
```diff
@@ -1,49 +1,157 @@
...
+}
+footer{
+    position: relative;
+    bottom: 0;
+    left: 0;
+    background-color:rgba(0, 0, 0, 0.4);
+    width: 100%;
+    height: 50px;
+    color: whitesmoke;
```
Comment (Regen1) [inline index.css]: Hex

File: index.php [hunk 5]
```diff
@@ -1,59 +1,59 @@
...
-            Use the respective forums if you need to reach our staff.</p>
-        </div>
-    </footer>
-</div>
-
+    <div class="wrapper">
+    <nav>
+    
+       <ul>
+           <li id ="logo"><a  href="index.html"><img src="Images/Paradise2icon.PNG" alt="server logo"></a></li>
```
Comment (Regen1) [inline index.php]: Formatting, not on the same line as everything else

File: index.php [hunk 6]
```diff
@@ -1,59 +1,59 @@
...
+                </li>
+                <li>
+                    <h2>Github</h2>
+                    <p>Space station 13 is an open source game.This means anyone can contribute. Most big servers run their own variant of the game, with distinct and unique elements to their codebase. We too, have our own version of the game, made possible through the hard work of our volunteer coders.</p> <p>You can view and contribute to our codebase <a href="https://github.com/ParadiseSS13/Paradise">at our github page.</a></p>
+                </li>
+            </ul>
+       
+   </main>
+   </div>
+   <footer><p>Licences, copyright and other guff like icons for patreon/discord/git goes here.</p></footer>
```
Comment (Regen1) [inline index.php]: Should add something about the MIT license here

**Review by Regen1 — CHANGES_REQUESTED** [review]
Only other thing that needs changing is perhaps the text, we need to finalize it and clear it with Freestyle

File: index.php [hunk 7]
```diff
@@ -1,59 +1,59 @@
...
+    </nav>
+    <main>
+        <h1>Paradise Station, a ss13 server</h1>
+        <div id="intro">
+            <p>
+                Paradise Station is a Space Station 13 server.
+                No idea what space station 13 is? Check out our new player guide <a href="https://nanotrasen.se/wiki/index.php/Guide_for_beginners"> here!</a>
+            </p>
+            <p>
+                We like to pride ourselves that the general playstyle on our server is a healthy mix or RP & action. So join us on the cyberiad, and try to stay alive, atleast for a while.
```
Comment (FreeStylaLT) [inline index.php]: We like to pride ourselves that the general playstyle on our server is a healthy mix of RP & Action. Come join us on the Cyberiad, and try to stay alive, at least for a while.

File: index.php [hunk 8]
```diff
@@ -1,59 +1,59 @@
...
+            </p>
+       </div>
+        <ul>
+                <li>
+                   <h2>Forums & discord</h2>
+                   <p>Throughout the years, we've built a solid community around our server, with an active forum and a public discord server.</p> <p> The <a href="https://nanotrasen.se/phpBB3/">forums</a> are our official discussion space, where everthing can be document easily, while the <a href="https://discord.gg/nuqD478">discord</a> is a more informal space where players and admins can hang out.</p>
```
Comment (FreeStylaLT) [inline index.php]: Throughout the years, we've built a solid community around our server, with an active forum and a public discord server.</p> <p> The <a href="https://nanotrasen.se/phpBB3/">forums</a> are our official discussion space, where everything can be documented easily, while the <a href="https://discord.gg/nuqD478">discord</a> is a more informal space where players and admins alike can hang out and chat.</p>

File: index.php [hunk 9]
```diff
@@ -1,59 +1,59 @@
...
+                </li>
+                <li>
+                   <h2>Patreon</h2>
+                   <p>Running a server costs money sadly. Previously we accepted donations to keep that server running. Now we've set up a patreon for that purpose. It also allows us to give thanks to those who make it possible.</p><p>You can find out more about the patron tiers on our <a href="https://www.patreon.com/ParadiseStation">Patreon</a> page!</p>
```
Comment (FreeStylaLT) [inline index.php]: Sadly, running a server is not free. Previously we accepted donations to keep that server running. Now we've set up a Patreon for that purpose. It also allows us to provide various cosmetic bonuses as a way of thanking our patrons.</p><p>You can find out more about the patron tiers on our <a href="https://www.patreon.com/ParadiseStation">Patreon</a> page!

File: index.php (hunk 8 above)
Comment (tigercat2000) [reply index.php]: Please make this multi-line as well, it's difficult to change it as-is.

Comment (JorisKeir) [reply index.php]: Seperate lines for the discord and the forum links?

Comment (tigercat2000) [reply index.php]: @Draconic-saint It's HTML, and you're using `<p>` tags anyways, newlines won't have any effect on the final product (I think)

**Review by Regen1 — APPROVED** [review]

**Pull request merged**